src/main/java/org/opensearch/security/dlic/rest/api/SecuritySSLCertsAction.java

```diff
@@ -136,6 +128,10 @@ protected void handleApiRequest(final RestChannel channel, final RestRequest req
                     );
                     return;
                 }
+                if (!restApiAdminPrivilegesEvaluator.isCurrentUserRestApiAdminFor(getEndpoint(), "reloadcerts")) {
```

Comment:
I would prefer the permissions check stay where it is at.  I only think accounts that have permissions to make the API call should be able to see if the configuration value was set one way or another.  What do you think?

Reply:
Good point will add debug message

Reply:
Could you move this to before the !certificatesReloadEnabled check?

Reply:
for sure.

Reply:
fixed it